### PR TITLE
clarity between instance snaphosts vs application snapshots

### PR DESCRIPTION
**Pull request by jgruica**
Screen Shot 2020-11-13 at 1 24 09 PM
Screen Shot 2020-11-13 at 1 48 30 PM

### CODE REVIEW

**Review by GraysonNull — APPROVED** [review]
Can we move the icon up a bit to be top aligned to the header text. Otherwise code looks good. Waiting on copy approval from @markpundsack

**Comment by jgruica** [comment]
> Can we move the icon up a bit to be top aligned to the header text. Otherwise code looks good. Waiting on copy approval from @markpundsack

updated!

**Comment by markpundsack** [comment]
LGTM. I'd like review by @salahalsaleh or @divolgin on the wording to make sure it's technically correct. Like "manifests" (are there multiple manifests, or a single manifest of multiple objects?)

**Review by divolgin — APPROVED** [review]

**Review by markpundsack — APPROVED** [review]

**Pull request merged**
